src/worksheet.rs

```diff
@@ -57,27 +58,10 @@ impl Into<libxlsxwriter_sys::lxw_image_options> for &ImageOptions {
             y_offset: self.y_offset,
             x_scale: self.x_scale,
             y_scale: self.y_scale,
-            row: 0,
-            col: 0,
-            filename: std::ptr::null_mut(),
+            object_position: 0,
```

Comment:
ref: https://libxlsxwriter.github.io/worksheet_8h.html#a915c52f231594642bad90422c5b2cf81
Hardcoded to option 0, LXW_OBJECT_POSITION_DEFAULT, for the time being.